draftfast/optimize.py

```diff
@@ -22,12 +22,27 @@ def run(rule_set: RuleSet,
         player_pool,
         player_settings,
     )
+
+    if rule_set.game_type == 'showdown':
+        if not optimizer_settings.showdown_teams or \
+                len(optimizer_settings.showdown_teams) != 2:
+            raise Exception(
```

Comment:
Why is this necessary? Couldn't the optimizer just pick this up from the player pool?